addons/news-annoucements/userscript.js

```diff
@@ -0,0 +1,3 @@
+export default async function ({ addon, global, console }) {
```

Comment:
I don't think you need to declare `addon, global, console` as we don't actually use it in the addon itself.

Reply:
I would leave it as is, just because better safe then sorry

Reply:
You can remove it, but many userscripts do this even if they don't need the objects.

Reply:
I'll remove it then.